lib/compat/experimental/blocks.php

```diff
@@ -52,6 +57,11 @@ function build_comment_query_vars_from_block( $block ) {
 			}
 		}
 
+		$order = ! empty( $block->context['comments/order'] ) ? $block->context['comments/order'] : null;
```

Comment:
What's the reason that the logic related to `order` is split into two chunks? Can we consolidate it in one place? When the context is set, we don't need to call `get_option` at all. Unless I'm missing something.

Reply:
In my testing, when I have the `inherit` attribute on the Comments Query Loop block enabled, the context value still gets processed here. It means that either the attribute should be set to `null` in the wrapping block or the check should take `inherit` into account.